pkgs/development/python-modules/pysvn/default.nix

```
@@ -8,52 +9,70 @@
 
 buildPythonPackage rec {
   pname = "pysvn";
-  version = "1.8.0";
-  disabled = isPy3k;
+  version = "1.9.12";
   format = "other";
 
   src = fetchurl {
     url = "http://pysvn.barrys-emacs.org/source_kits/${pname}-${version}.tar.gz";
-    sha256 = "0srjr2qgxfs69p65d9vvdib2lc142x10w8afbbdrqs7dhi46yn9r";
+    sha256 = "sRPa4wNyjDmGdF1gTOgLS0pnrdyZwkkH4/9UCdh/R9Q=";
   };
 
-  buildInputs = [ pkgs.subversion pkgs.apr pkgs.aprutil pkgs.expat pkgs.neon pkgs.openssl ]
-    ++ (if stdenv.isLinux then [pkgs.e2fsprogs] else []);
+  buildInputs = with pkgs; [ bash subversion apr aprutil expat neon openssl ]
+    ++ (if stdenv.isLinux then [ pkgs.e2fsprogs ] else []);
```

```suggestion
    ++ optionals stdenv.isLinux [ pkgs.e2fsprogs ];
```
Is that pkgs pass in really required? In general it should be avoided.

---

It was written that way, I'll check if I can improve it